portality/formcontext/formcontext.py

```diff
@@ -1438,8 +1438,21 @@ def _set_choices(self):
         pass
     """
 
-### Journal form contexts ###
+class ManEdArticleReview(PrivateContext):
```

Comment:
It looks like you started out with this approach and then abandoned it, but I think it has merit, because it's exactly what the formcontexts were designed for.  The parent class `Private Context` is heavily geared towards applications and journals, but you would get away with using it for articles too, I think.